lpa-submissions-e2e-tests/cypress/integration/accuracyAppellantsSubmission/accuracyAppellantsSubmission.step.js

```diff
@@ -77,8 +82,16 @@ Then('the radio group label is {string}', (label) => {
 
 Then('the user is shown the error message {string}', (errorMessage) => {
   errorMessage === 'Select yes if the information accurately reflects the planning application'
-    ? cy.validateErrorMessage(errorMessage, '[data-cy="accurate-submission-error"]', 'accurate-submission')
-    : cy.validateErrorMessage(errorMessage, '[data-cy="inaccuracy-reason-error"]', 'inaccuracy-reason');
+    ? cy.validateErrorMessage(
+        errorMessage,
+        '[data-cy="accurate-submission-error"]',
+        'accurate-submission',
+      )
+    : cy.validateErrorMessage(
+        errorMessage,
+        '[data-cy="inaccuracy-reason-error"]',
+        'inaccuracy-reason',
+      );
 });
```

Comment:
i do like me a nice ternary however...
when you're writing step defs remember that clarity is probably -the- most important thing. i can pick my way through that lovely single-functionally-complete-line.. can Ravi? Can Karuna? because they need to be able to read and understand these files as well..

where possible, at least abstracting things out into methods with meaningful names has value in these stepdefs.. you need them to be a place that a normal human has a fighting chance of reading it and saying "yes, this is testing the things i think it needs to test"

(i'm now braced for the 'cypress best practices' link that you followed to do this ;) but you get what i mean? functionally splendid; somewhat opaque to mortals?)

Reply:
That's not code I brought in - I just formatted it. A conversation to have outside of this review I think. Shows we need to review the testing refactor carefully